ci-operator/config/openshift-qe/ocpqe-security-tools/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- cpmeadors
+- asood-rh
+- paigerube14
+reviewers:
+- cpmeadors
+- asood-rh
+- paigerube14
```

Comment:
nit : We may want to add new lines to owners files to take care of this for final commit.